core/openai.go

```diff
@@ -0,0 +1,110 @@
+package core
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+)
+
+var	inferURL = os.Getenv("INFER_URL")
```

Comment:
**(issue)** No need to leave it as configuration. The package shall absorb this.

```suggestion
const inferURL = https://api.openai.com/v1/
```